### DISCUSSION TITLE
Welcome to Alexa-Weather-Skill Discussions!

### DISCUSSION THREAD

**Discussion by jeffthomps** [jeffthomps/Alexa-Weather-Skill#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeffthomps** [comment]
It looks like the link to the forum is no longer working. I will look into updating that if possible.

**Comment by wariac** [comment]
Hi, is this skill still working?  Trying to get it working with my Dads Davis WeatherLink.

thanks
Paul
